config/db

```diff
@@ -50,36 +49,36 @@ rbx_1.2.3_patch_level=20110315
 rbx_1.2.4_patch_level=20110705
 rbx_url=http://asset.rubini.us
 rbx_2.0.0_url=http://releases.rubini.us
```

Comment:
I'd love to change this to https://s3.amazonaws.com/releases.rubini.us (since releases.rubini.us just points to the S3 bucket anyway). However, wasn't sure if that was considered kosher.